quickstart/src/selectmedia.js

```diff
@@ -15,8 +15,72 @@ const localTracks = {
  * @returns {Promise<void>} Promise that is resolved if successful
  */
 async function applyInputDevice(kind, deviceId, render) {
+  if(!window.CitrixWebRTC) {
+    throw new Error('CitrixWebRTC is not available');
+  }
   // Create a new LocalTrack from the given Device ID.
-  const [track] = await createLocalTracks({ [kind]: { deviceId } });
+  const [track] = await createLocalTracks({
+    [kind]: { deviceId },
+    RTCPeerConnection: CitrixWebRTC.CitrixPeerConnection.bind(window.CitrixWebRTC),
+    getUserMedia: (config) => window.CitrixWebRTC.getUserMedia(config),
+    enumerateDevices: window.CitrixWebRTC.enumerateDevices,
+    MediaStream: class CustomMediaStream extends MediaStream {
```

Comment:
Same, why override MediaStream?

Reply:
We've already sorted out this question, but we need this for adding non-native tracks to the MediaStream object.